service/history/decision/handler.go

```diff
@@ -730,6 +730,8 @@ func (handler *handlerImpl) handleBufferedQueries(
 		buffered := queryRegistry.GetBufferedIDs()
 		handler.logger.Info(
 			"failing query because worker does not support consistent query",
+			tag.Dynamic("clientImpl", clientImpl),
+			tag.Dynamic("clientFeatureVersion", clientFeatureVersion),
```

Comment:
did you want to make this a proper tag maybe? also I think we use kebab-case with logs `client-feature-version` maybe a string tag?